config/src/config/network_config.rs

```diff
@@ -43,7 +43,7 @@ pub const CONNECTIVITY_CHECK_INTERVAL_MS: u64 = 5000;
 pub const MAX_CONCURRENT_NETWORK_REQS: usize = 100;
 pub const MAX_CONNECTION_DELAY_MS: u64 = 60_000; /* 1 minute */
 pub const MAX_FULLNODE_OUTBOUND_CONNECTIONS: usize = 4;
-pub const MAX_INBOUND_CONNECTIONS: usize = 30; /* At 5k TPS this could easily hit ~50MiB a second */
+pub const MAX_INBOUND_CONNECTIONS: usize = 100;
```

Comment:
changing the value seems weirdly out of scope for the rest of the PR

Reply:
Aah, it was because these values were being overwritten by various config files (e.g., in the docker and terraform directories). This removes the overrides. Technically, it could be a separate commit (but, it's still part of the goal 😄).